tasks/agent.py

```diff
@@ -159,6 +159,8 @@ def build(
     """
     flavor = AgentFlavor[flavor]
 
+    warn_deprecated_parameter(python_runtimes, '--python-runtimes')
```

Comment:
What is the plan for this deprecation? I wonder whether we shouldn't just remove it if nothing inside the repo references it.

Reply:
I have no strong opinion on the matter, the intent was to let anyone relying on local scripts/their shell history when invoking `*.build` tasks to adjust before breaking, but we can remove it directly altogether

Reply:
They are now gone